docs/pages/access-controls/introduction.mdx

```diff
@@ -1,33 +1,21 @@
 ---
 title: Access Controls Introduction
-description: Provide Role Based Access Controls for SSH, Kubernetes, Databases and Web Apps
+description: Provide Role-Based Access Controls (RBAC) for SSH, Kubernetes, Databases, and Web Apps.
```

Comment:
I think you started changing to Role-Based Access Control (without s) everywhere, which I think is correct, so let's change here too.